views/events/event.jade

```diff
@@ -56,8 +56,12 @@ block content
 
             for comment in comments
                 div.row.row-bottom-padding.comment-content
-                    +profile-picture(comment.authorObject, 'participant-avatar')
-                    span.participant-heading= comment.authorObject.profile.name
+                    if comment.authorObject
+                        +profile-picture(comment.authorObject, 'participant-avatar')
+                        span.participant-heading= comment.authorObject.profile.name
+                    else

```

Comment:
This fixes a bug causing hack pages to crash if they contained comments before my last commit.